examples/linux.yaml

```
@@ -100,10 +101,14 @@ self:
     backup:
         extended-status: true
         source: ./
+        # schedule:
+        #   at: "*:15"
+        #   permission: system
     copy:
         initialize: true
         schedule-permission: user
         schedule:
+          - "*:15"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)